src/storage/cdb/cdb_ffi/src/lib.rs

```diff
@@ -11,63 +12,102 @@ mod ccommon;
 use std::convert::From;
 use std::ffi::CStr;
 use std::os::raw::c_char;
+use std::path::PathBuf;
 use std::ptr;
 
 use cdb_rs::cdb;
-use cdb_rs::{Result, CDB};
+use cdb_rs::{Mmap, Result, CDB};
```

Comment:
Does Rust have style guide on the ordering of imports?

Reply:
well, intellij's plugin does them strict alphabetical both linewise and inside the curlies.

Here's the style guide entry: https://github.com/rust-lang-nursery/fmt-rfcs/blob/master/guide/items.md